src/panel/poi/ActionButtons.jsx

```diff
@@ -17,6 +17,10 @@ const ActionButtons = ({
     return handler(e);
   };
 
+  const onShare = target => {
+    Telemetry.add(Telemetry.POI_SHARE_TO, { target, id: poi.id });
```

Comment:
do we need to specify the poi.id ?

Reply:
Maybe not. I'm 100% sure it will be asked in the future, but let's remove them for now.